packages/node/src/configure/SubqueryProject.ts

```diff
@@ -73,7 +69,7 @@ export class SubqueryProject implements ISubqueryProject {
     this.#dataSources = await insertBlockFiltersCronSchedules(
       this.dataSources,
       getTimestamp,
-      isRuntimeDs,
+      isRuntimeDs, // TODO need update main repo to support this
```

Comment:
I don't understand this comment, what needs to change?

Reply:
The subql repo needs to be released first, then we can update the dependencies on our end.